src/components/ItaliaTheme/Blocks/IconBlocks/Edit.jsx

```diff
@@ -63,7 +63,7 @@ class Edit extends SubblocksEdit {
               className="background-image"
               style={{
                 backgroundImage: `url(${
-                  this.props.data.background[0]?.image?.download ??
+                  this.props.data.background[0]?.image?.scales.huge.download ??
```

Comment:
mettiamo la condizione se esiste huge, quindi:
this.props.data.background[0]?.image?.scales?.huge?.download